modules/openy_gc_auth/modules/openy_gc_auth_yusa/src/Form/VirtualYUSALoginForm.php

```diff
@@ -280,4 +289,27 @@ protected function sendEmailVerification($user, $provider_config, $mail) {
     $this->privateTempStore->set($mail, TRUE);
   }
 
+  /**
+   * Defines if we need to verify the user in a current browser.
+   *
+   * @param \Drupal\user\Entity\User $user
+   *   User object.
+   *
+   * @return bool
+   *   Check result.
+   */
+  protected function isVerificationNeeded(User $user): bool {
```

Comment:
We have 2 places with equal code.
Can we store it at one place?

Reply:
There is a lot of alike code in the login forms of different providers. Can we plan a refactoring to create some base form or smth like this?